share/jupyter/nbconvert/templates/reveal/index.html.j2

```diff
@@ -39,7 +39,6 @@
 
 <!-- General and theme style sheets -->
 <link rel="stylesheet" href="{{ reveal_url_prefix }}/dist/reveal.css">
```

Comment:
Can we move this line down too? It seems I get better results (e.g., centered slide titles) making this reveal.css also take precedence over the theming below.